ygot/struct_validation_map.go

```diff
@@ -475,14 +475,33 @@ func MergeJSON(a, b map[string]interface{}) (map[string]interface{}, error) {
 	return o, nil
 }
 
+// MergeOpt is an interface that is implemented by the options to the MergeStructInto
```

Comment:
Suggest to change this to say "options to the MergeStructs and MergeStructInto functions", since this is now propagated to both implementations.